### ISSUE TITLE
Training Problem

### ISSUE THREAD

**Issue by huunhan312** [azmathmoosa/azFace#3]
Dear azmathmoosa,

I find that your training is good for one object. So I want to train your configuration on my custom object. However, I got problem with NAN of average loss.

I train by this way:
* use your configuration with two classes by setting class = 2 and previous layer output of (2+5)*5 = 35
* Use pre-trained weight factors from darknet reference (https://pjreddie.com/darknet/imagenet/#reference)

I tested with batch size of 32, 64 and learning rate 0.001, 0.0005. After about 50 iterations, I got NAN average loss.
I do the same thing but with yolov2-tiny, yolov3-tiny (of course, tuning from published weights), darknet can train well.

Could you share your ideas about this problem? And Did you do any special steps to train your model for face detection.

Thank you

**Comment by huunhan312** [comment]
Oh, maybe I got the solution now. We need to make the weight factors only for needed convolution layers. I used partial parameter to make that weight factors and it works now.
